### ISSUE TITLE
Windows + http2 --> java.lang.NoClassDefFoundError: org/eclipse/jetty/alpn/ALPN$Provider

### ISSUE THREAD

**Issue by jschneider** [jooby-project/jooby#551]
When starting the server with http2() and securePort() enabled, we run into:

java.lang.NoClassDefFoundError: org/eclipse/jetty/alpn/ALPN$Provider
  at io.netty.handler.ssl.JdkAlpnApplicationProtocolNegotiator$1.<init>(JdkAlpnApplicationProtocolNegotiator.java:26)
  at io.netty.handler.ssl.JdkAlpnApplicationProtocolNegotiator.<clinit>(JdkAlpnApplicationProtocolNegotiator.java:24)
  at io.netty.handler.ssl.JdkSslContext.toNegotiator(JdkSslContext.java:277)
  at io.netty.handler.ssl.JdkSslServerContext.<init>(JdkSslServerContext.java:226)
  at io.netty.handler.ssl.SslContext.newServerContextInternal(SslContext.java:407)
  at io.netty.handler.ssl.SslContextBuilder.build(SslContextBuilder.java:393)
  at org.jooby.internal.netty.NettySslContext.build(NettySslContext.java:68)  
  at org.jooby.internal.netty.NettyServer.start(NettyServer.java:105)
  at org.jooby.Jooby.start(Jooby.java:3673)
  at org.jooby.Jooby.start(Jooby.java:3630)
  at org.jooby.Jooby.run(Jooby.java:3523)
  at de.avat.service.pilot.Selene.main(Selene.java:158)
  at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at java.lang.reflect.Method.invoke(Method.java:498)
  at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.alpn.ALPN$Provider
  at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
  at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
  at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
  at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
  ... 17 common frames omitted

It is working with Linux.
When debugging I could find a difference in:
 org.jooby.internal.netty.NettySslContext:60

Under Windows "OpenSsl.isAlpnSupported()" returns false, under Linux true

**Comment by jknack** [comment]
Add this dependency and try again please:

```xml
<dependency>
          <groupId>io.netty</groupId>
          <artifactId>netty-tcnative-boringssl-static</artifactId>
          <version>1.1.33.Fork19</version>
          <classifier>windows-x86_64</classifier>
        </dependency>
```

**Comment by jknack** [comment]
I tried in Windows 8 IE11 + Jooby 1.0.0 Netty.

Everything works as expected.

Closing this issue as invalid.

**Comment by jschneider** [comment]
Run into this issue again on another project. This is definitely not fixed in 1.1.0 on Linux.
